test/lib/e2e/IExecOrderModule.test.js

```diff
@@ -323,6 +328,143 @@ describe('order', () => {
     });
   });
 
+  describe('estimateMatchOrders()', () => {
+    let iexecProvider;
+    let apporderTemplate;
+    let datasetorderTemplate;
+    let workerpoolorderTemplate;
+    let voucherTypeId;
+
+    beforeAll(async () => {
+      const providerConfig = getTestConfig(iexecTestChain)();
+      iexecProvider = providerConfig.iexec;
+      apporderTemplate = await deployAndGetApporder(iexecProvider, {
+        volume: 10,
+        appprice: 5,
+      });
+      datasetorderTemplate = await deployAndGetDatasetorder(iexecProvider, {
+        volume: 7,
+        datasetprice: 1,
+      });
+      workerpoolorderTemplate = await deployAndGetWorkerpoolorder(
+        iexecProvider,
+        { volume: 5, workerpoolprice: 1 },
+      );
+      voucherTypeId = await createVoucherType(iexecTestChain)({
+        description: 'test voucher type',
+        duration: 42,
```

Comment:
42s may be too short as the voucher duration for the test (60 * 60 will be safe)